docs/statistics.md

```diff
@@ -1,269 +1,425 @@
-Refer to [SRT Packet Filtering & FEC](packet-filtering-and-fec.md).
+The total number of lost packets that were not covered by the packet filter (refer to [SRT Packet Filtering & FEC](packet-filtering-and-fec.md)). Receiver only.
 
-Introduced in v1.4.0.
+Introduced in SRT v1.4.0.
 
-## byteSentTotal
+### byteSentTotal
 
-Same as `pktSentTotal`, but expressed in bytes, including payload and all headers (SRT+UDP+IP). \
-20 bytes IPv4 + 8 bytes of UDP + 16 bytes SRT header. Sender side.
+Same as `pktSentTotal`, but expressed in bytes, including payload and all the headers (20 bytes IPv4 + 8 bytes UDP + 16 bytes SRT). Sender side.
 
-## byteRecvTotal
+### byteRecvTotal
 
-Same as `pktRecvTotal`, but expressed in bytes, including payload and all headers (SRT+UDP+IP). \
-20 bytes IPv4 + 8 bytes of UDP + 16 bytes SRT header. Receiver side.
+Same as `pktRecvTotal`, but expressed in bytes, including payload and all the headers (20 bytes IPv4 + 8 bytes UDP + 16 bytes SRT). Receiver side.
 
-## byteRcvLossTotal
+### byteRcvLossTotal
 
-Same as `pktRcvLossTotal`, but expressed in bytes, including payload and all headers (SRT+UDP+IP). \
-20 bytes IPv4 + 8 bytes of UDP + 16 bytes SRT header. Receiver side.
+Same as `pktRcvLossTotal`, but expressed in bytes, including payload and all the headers (20 bytes IPv4 + 8 bytes UDP + 16 bytes SRT). Receiver side.
 
-## byteRetransTotal
+### byteRetransTotal
 
-Same as `pktRetransTotal`, but expressed in bytes, including payload and all headers (SRT+UDP+IP). \
-20 bytes IPv4 + 8 bytes of UDP + 16 bytes SRT header. Sender side only.
+Same as `pktRetransTotal`, but expressed in bytes, including payload and all the headers (20 bytes IPv4 + 8 bytes UDP + 16 bytes SRT). Sender side.
 
-## byteSndDropTotal
+### byteSndDropTotal
 
-Same as `pktSndDropTotal`, but expressed in bytes, including payload and all headers (SRT+UDP+IP). \
-20 bytes IPv4 + 8 bytes of UDP + 16 bytes SRT header. Sender side only.
+Same as `pktSndDropTotal`, but expressed in bytes, including payload and all the headers (20 bytes IPv4 + 8 bytes UDP + 16 bytes SRT). Sender side.
 
-## byteRcvDropTotal
+### byteRcvDropTotal
 
-Same as `pktRcvDropTotal`, but expressed in bytes, including payload and all headers (SRT+UDP+IP). \
-Bytes for dropped packet payloads are estimated based on average packet size. \
-20 bytes IPv4 + 8 bytes of UDP + 16 bytes SRT header. Receiver side only. 
+Same as `pktRcvDropTotal`, but expressed in bytes, including payload and all the headers (20 bytes IPv4 + 8 bytes UDP + 16 bytes SRT). Bytes for the dropped packets' payloads are estimated based on the average packet size. Receiver side.
 
-## byteRcvUndecryptTotal
+TODO: Bytes for the dropped packets' payloads are estimated based on the average packet size. - What's about the other stats from this group? -> applicable for byteRcvLossTotal (as of now it has lost and out-of-order and undecrypted, but we do not size for lost and potentially out of order)
```

Comment:
This should be mentioned in every stats which is based on the number of bytes of packets that have never arrived (even late).

Reply:
What's exactly mentioned in my TODO except you have other suggestions than byteRcvLossTotal.